destination/writer/writer.go

```diff
@@ -580,7 +586,7 @@ func (s *SnowflakeCSV) Merge(
 
 		rowsAffected, err := res.RowsAffected()
 		if err != nil {
-			sdk.Logger(ctx).Err(err).Msgf("could not determine rows affected on merge into table %s from %s", s.TableName, updatesFilename)
+			sdk.Logger(ctx).Err(err).Msgf("could not determine rows affected on merge into table %s from %s", s.TableName, updatesFilename) //nolint:lll // formatted message
```

Comment:
Not sure if this solves the line length issue, but we could try breaking the calls into separate lines.
```suggestion
			sdk.Logger(ctx).
				Err(err).
				Msgf(
					"could not determine rows affected on merge into table %s from %s",
					s.TableName,
					updatesFilename,
				)
```